yt_dlp/extractor/ard.py

```diff
@@ -455,6 +473,12 @@ def _real_extract(self, url):
             'subtitles': subtitles,
             'is_live': is_live,
             'age_limit': age_limit,
+            **traverse_obj(media_data, {
+                'chapters': ('pluginData', 'jumpmarks@all', 'chapterArray', ..., {
```

Comment:
Actually, let's ensure the start time is present.
```suggestion
                'chapters': ('pluginData', 'jumpmarks@all', 'chapterArray', lambda _, v: int_or_none(v['chapterTime']), {
```